tests/queries/0_stateless/00087_math_functions.sql

```diff
@@ -137,3 +137,10 @@ select erf(10) = 1;
 select erfc(0) = 1;
 select erfc(-10) = 2;
 select erfc(28) = 0;
+
+select factorial(-1) = 1;
```

Comment:
Better to create new test, for us it will be easier to track if test becomes flacky.